osf/models/mixins.py

```diff
@@ -2108,10 +2108,13 @@ def _check_spam_user(self, user):
         ):
             self.suspend_spam_user(user)
 
-    def suspend_spam_user(self, user):
-        if user.spam_status == SpamStatus.HAM:
+    def suspend_spam_user(self, user, set_spam=False):
+        if user.spam_status == SpamStatus.HAM and not set_spam:
```

Comment:
The `set_spam` arg seems overloaded -- here it's behaving as a `force`, below it's behaving as one would expect. If we want to make sure ham users don't accidentally get caught in the crossfire of this script, I don't think it should be used here.

Reply:
Not sure why I put that there. Thanks